GitUI/CommandsDialogs/FormResolveConflicts.cs

```diff
@@ -94,6 +99,21 @@ public partial class FormResolveConflicts : GitModuleForm
         #endregion
 
         private readonly IFullPathResolver _fullPathResolver;
+        private enum SolveMergeConflictActionType
+        {
+            atNone = 0,
+            atKeepLocal = 1,
+            atKeepRemote = 2,
+            atKeepBase = 3
+        }
+
+        private SolveMergeConflictActionType _solveMergeConflictDialogResult;
+        private bool _SolveMergeConflictApplyToAll;
```

Comment:
Here and below, fields must follow the _camelCase_ notation
```suggestion
        private bool _solveMergeConflictApplyToAll;
```